syft/frameworks/torch/hook/hook.py

```diff
@@ -141,7 +141,8 @@ def __init__(
         self._hook_native_tensor(torch.Tensor, TorchTensor)
 
         if dependency_check.crypten_available:
```

Comment:
Q: Should this be removed?

Reply:
Oh, I shouldn't have pushed that, I was only commenting this to not hook mpc tensor to get cat and stuff working, will just re-enable it for the moment